### ISSUE TITLE
Site navigation sidebar style

### ISSUE THREAD

**Issue by najuzilu** [ExecutableBookProject/quantecon-mini-example#16]
In a previous version, the title under the sidebar _Scientific Python QuickStart_ had a different font size from its subcategories to indicate the structure of the toctree. If I remember correctly it was the same font size as the header key. This is not the case for our current example:

<img src="https://user-images.githubusercontent.com/33075058/80561223-193ce400-8a27-11ea-846e-816c3566495b.png"  height="350">

I also find it strange that `index.md` shows up on the sidebar. I would expect that if the user wants to go back to the landing page, clicking on the logo should take them there.

@jstac do you have any thoughts on the above?

**Comment by jstac** [comment]
Thanks @najuzilu.  I agree about logo directing to the landing page.

Regarding the sidebar, everything in that side TOC is at the same level, so I would expect the font sizes to be the same.  Is that different from what you would expect?

**Comment by najuzilu** [comment]
I would expect the index file to have a different font size because it's one level up from the rest of the files. So not all files are at the same level. Does this make sense?

```
- file: docs/index
  sections:
    - file: docs/about_py
    - file: docs/getting_started
    - file: docs/python_by_example
    - file: docs/learn_more
```

**Comment by jstac** [comment]
Ah, now I understand.  I agree.

**Comment by choldgraf** [comment]
Check out the `use_homepage_in_navbar` option: https://jupyterbook.org/customize/config.html#configuration-reference

we really need to improve the documentation for that configuration page...

**Comment by najuzilu** [comment]
@choldgraf thank you for pointing this out. I agree with you regarding the documentation - it's growing rapidly.

**Comment by najuzilu** [comment]
I will close this issue since it has been fixed by PR #21.
